setup.py

```diff
@@ -22,6 +26,8 @@
     'packages': find_packages(),
     'license': "Apache License 2.0",
     'install_requires': list(open('requirements.txt')),
+    'extras_require': extras_requires,
+    'test_suite': 'test',
```

Comment:
`test` in setuptools is deprecated. Refer to [this](https://setuptools.readthedocs.io/en/latest/userguide/commands.html?highlight=test_suite#test-build-package-and-run-a-unittest-suite).